scripts/export-chart-version.sh

```diff
@@ -9,4 +9,5 @@ rel_dir=$( dirname $script_dir )
 chart_dir=$rel_dir/$1
 
 cd $chart_dir
-yq r Chart.yaml version
+# Strip off any "-next" suffix
+yq r Chart.yaml version | sed 's/\(.*\)-next/\1/'
```

Comment:
I don't think this should be stripping next - this script should export the version as-is. Since the stripping is for the release, we should do it in make-release.sh.

Reply:
I really didn't like that change either.  Fixed.  Other things thus simplified.